### ISSUE TITLE
fix incompatibility with sphinx 1.8

### ISSUE THREAD

**Issue by dimpase** [sagemath/sagenb#465]
See https://trac.sagemath.org/ticket/26451 - one gets
`ApplicationError: Source directory and destination directory cannot be identical`

**Comment by dimpase** [comment]
This particular problem is not something that would occur regularly. Anything that does the job is welcome, anyway.

**Comment by jhpalmieri** [comment]
The current version of sphinxify in sagenb says
```
    if not SAGE_DOC_SRC and not os.path.exists(confdir):
        # If we don't have Sage, we need to do our own configuration
        # This may be inefficient or broken.  TODO: Find a faster way to do this.
        temp_confdir = True
        confdir = mkdtemp()
        generate_configuration(confdir)
```
Can we always assume that we "have Sage"? If so, we can probably replace most of the file with `from sage.misc.sphinxify import sphinxify`. Keep `is_sphinx_markup` and get rid of everything else.

**Comment by dimpase** [comment]
fixed by #466.
